### ISSUE TITLE
Incorrect log for Complex types

### ISSUE THREAD

**Issue by tarunvd** [bilal-fazlani/tracker-enabled-dbcontext#122]
I have just started using the tracker enabled dbcontext library and have run into an issue with complex types. I have a complex entity inside another entity, but any changes to the values of the complex type do not get logged. Instead, the type of the complex type gets logged in `AuditLogDetails`

```
public class Address : ICloneable
    {
        public string AddressLine1 { get; set; }

        public string Town { get; set; }

        public string PostCode { get; set; }
    }

public class Employee
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public Address Address { get; set; }
    }

```

So, `Address` is not a db entity and does not have a DbSet in the context. The fields generated via EF in the Sql tables are like `Address_AddressLine1`, `Address_Town`, `Address_PostCode`, etc. These are correctly saved via EF to the corresponding table fields when any changes to Employee.Address are done. 

I am using tracking via DataAnnotations and all fields apart from the `Address` field are being logged correctly. However, `AuditLogDetails` has only 1 row for any changes done in the `Address` and with the type of the complex type saved in the OriginalValue and NewValue:

```
PropertyName:  Address
OriginalValue: DataContext.ComplexTypes.Address
NewValue: DataContext.ComplexTypes.Address
```

@bilal-fazlani, Can the library be updated to handle this, ie. if the `NewValue/OriginalValue` being saved in the `AuditLogDetails` are object types, then instead of saving the type of the object, save the individual properties of the object as separate rows in `AuditLogDetails`. I believe this is a common use case and many projects would be using such complex type structures, so this needs to be resolved.

**Comment by theyo** [comment]
@bilal-fazlani This issue was resolved by #137.

**Comment by bilal-fazlani** [comment]
Closing issue. Fixed in [version 3.6.2-beta](https://www.nuget.org/packages/TrackerEnabledDbContext/3.6.2-beta) by #137.

**Comment by tarunvd** [comment]
@bilal-fazlani @theyo Thanks for looking at this.
